tasks/configure.yml

```diff
@@ -76,19 +74,28 @@
   notify:
     - restart go-server
 
+- name: Ensures go-server backup cron job
+  become: yes
+  cron:
+    hour: 9,12,15,18
+    job: "/home/{{ gocd_server.user }}/bin/aws_s3_backup.sh"
+    name: Backup go-server
+    minute: 0
+    weekday: 1-7
+
 - name: Setup AWS credentials
   become: yes
   become_user: "{{ gocd_server.user }}"
   template:
-    dest: "/home/go/.aws/credentials"
+    dest: "/home/{{ gocd_server.user }}/.aws/credentials"
```

Comment:
@racer1680 it's only when you do explicitly set `gocd_server.aws.access_key_id` - in cases where you want to spin up local VM and tests things that require access to aws apis.

In most cases there will be no AWS credentials file on the box.